strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -267,7 +262,7 @@ async def handle_async_results(
                     next_message = NextMessage(id=operation.id, payload=next_payload)
                     await operation.send_message(next_message)
         except asyncio.CancelledError:
-            # CancelledErrors are expected during task cleanup.
+            # Not a BaseException until 3.8.
```

Comment:
not sure what this comment means 😊

Reply:
Oh.  I'm commenting on the fact that we are catching an exception just to re-raise it.  In python 3.8 this code isn"t necessary because CancelledError is a `BaseException` and there is no BaseException clause.    But in 3.7, we need to catch and raise CancelledError so that it isn't caught by the `Exception`  clause.
Normally, CancelledErrors aren't caught at all in more recent python versions except in very rare cases.  Do you have a suggestion on how to better phrase the comment?